docs/es/docs/tutorial/path-params.md

```diff
@@ -46,20 +46,22 @@ Pero si abres tu navegador en <a href="http://127.0.0.1:8000/items/foo" class="e
 
 ```JSON
 {
-    "detail": [
-        {
-            "loc": [
-                "path",
-                "item_id"
-            ],
-            "msg": "value is not a valid integer",
-            "type": "type_error.integer"
-        }
-    ]
+  "detail": [
+    {
+      "type": "int_parsing",
+      "loc": [
+        "path",
+        "item_id"
+      ],
+      "msg": "Input should be a valid integer, unable to parse string as an integer",
+      "input": "foo",
+      "url": "https://errors.pydantic.dev/2.1/v/int_parsing"
+    }
+  ]
 }
 ```
 
-debido a que el parámetro de path `item_id` tenía el valor `"foo"`, que no es un `int`.
+debido a que el parámetro de <abbr title="también conocido en español como: ruta">path</abbr> `item_id` tenía el valor `"foo"`, que no es un `int`.
 
 El mismo error aparecería si pasaras un `float` en vez de un `int` como en: <a href="http://127.0.0.1:8000/items/4.2" class="external-link" target="_blank">http://127.0.0.1:8000/items/4.2</a>
 
```

Comment:
I suggest to replace following sentence:

- line: 73 `Esto es increíblemente útil cuando estás desarrollando y debugging código que interactúa con tu API.`
- **to:** `Esto es increíblemente útil durante el desarrollo y la depuración del código que interactúa con tu API.`